packages/tiptap/extensions.ts

```diff
@@ -19,6 +20,10 @@ export const tipTapExtensions: AnyExtension[] = [
       crossorigin: 'use-credentials'
     }
   }),
+  Markdown.configure({
+    html: false,
+    transformCopiedText: true
+  }),
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 10 to 26]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->I think we need an export for vue version to ensure its not gonna break<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/un/inbox/353?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->